test/issue-578/issue-578.test.ts

```diff
@@ -0,0 +1,11 @@
+import { assertEquals } from "https://deno.land/std@0.178.0/testing/asserts.ts";
+
+Deno.test("issue #478", async () => {
```

Comment:
typo here it should be "issue #578", doesn't matter if you don't want to change haha

Reply:
Whoops, good catch! That's an obvious copy and paste error on part. Corrected the reference to point to the right issue number. Thanks for reviewing!